### PR TITLE
add list of Employee field

### PR DESCRIPTION
**Pull request by kiurchv**

### CODE REVIEW

File: apps/graphql/test/web/resolvers/employee_resolver_test.exs [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    conn = put_scope(conn, "employee:read")
+
+    {:ok, %{conn: conn}}
+  end
+
+  describe "list" do
+    test "return all for NHS client", %{conn: conn} do
+      nhs()
+
+      for _ <- 1..2, do: insert(:prm, :employee)
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/employee_resolver_test.exs]:
```suggestion
      insert_list(2, :prm, :employee)
```

File: apps/graphql/test/web/resolvers/employee_resolver_test.exs [hunk 2]
```diff
@@ -0,0 +1,174 @@
+defmodule GraphQLWeb.EmployeeResolverTest do
+  @moduledoc false
+
+  use GraphQLWeb.ConnCase, async: true
+
+  import Core.Factories, only: [insert: 2, insert: 3]
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/employee_resolver_test.exs]:
```suggestion
  import Core.Factories, only: [insert: 2, insert: 3, insert_list: 3]
```

File: apps/graphql/test/web/resolvers/employee_resolver_test.exs [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+      resp_entities = get_in(resp_body, ~w(data employees nodes))
+
+      assert nil == resp_body["errors"]
+      assert 2 == length(resp_entities)
+    end
+
+    test "return only related for MSP client", %{conn: conn} do
+      msp()
+
+      legal_entities = for _ <- 1..2, do: insert(:prm, :legal_entity)
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/employee_resolver_test.exs]:
```suggestion
      legal_entities = insert_list(2, :prm, :legal_entity)
```

File: apps/graphql/test/web/resolvers/employee_resolver_test.exs [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+      assert nil == resp_body["errors"]
+      assert 1 == length(resp_entities)
+      assert related_employee.id == hd(resp_entities)["databaseId"]
+    end
+
+    test "return forbidden error for incorrect client type", %{conn: conn} do
+      mis()
+
+      for _ <- 1..2, do: insert(:prm, :employee)
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/employee_resolver_test.exs]:
```suggestion
      insert_list(2, :prm, :employee)
```

File: apps/core/test/support/factories.ex [hunk 5]
```diff
@@ -44,6 +44,10 @@ defmodule Core.Factories do
     |> repo_insert!(type)
   end
 
+  def insert_list(number_of_records, type, factory, attrs \\ []) do
```
Comment (PavelVesnin) [inline apps/core/test/support/factories.ex]: why not `insert_many`?

Comment (kiurchv) [reply apps/core/test/support/factories.ex]: ExMachina uses `build_list` name for similar function

Comment (kiurchv) [reply apps/core/test/support/factories.ex]: Also, there are available the `insert_many` helper function in the `ExMachina.Ecto` module

**Pull request merged**